src/utils/metrics.ts

```diff
@@ -290,6 +305,10 @@ export async function insertOss(
     data.updated_at = comment.updated_at;
     data.target_id = issue.number;
     data.target_type = 'issue';
+    data.product_area = issue.labels?.find((label) =>
+      label.name.startsWith(PRODUCT_AREA_LABEL_PREFIX)
```

Comment:
nit: would be nice to add coverage for this.

Reply:
done